docs/modules/sort/datastream_example.md

```diff
@@ -6,1716 +6,149 @@ sidebar_position: 3
-                                "type":"base",
-                                "name":"id",
-                                "formatInfo":{
-                                    "type":"long"
-                                }
-                            }
-                        },
-                        {
-                            "type":"fieldRelationShip",
-                            "inputField":{
-                                "type":"base",
-                                "name":"name",
-                                "formatInfo":{
-                                    "type":"string"
-                                }
-                            },
-                            "outputField":{
-                                "type":"base",
-                                "name":"name",
-                                "formatInfo":{
-                                    "type":"string"
-                                }
-                            }
-                        },
-                        {
-                            "type":"fieldRelationShip",
-                            "inputField":{
-                                "type":"base",
-                                "name":"age",
-                                "formatInfo":{
-                                    "type":"int"
-                                }
-                            },
-                            "outputField":{
-                                "type":"base",
-                                "name":"age",
-                                "formatInfo":{
-                                    "type":"int"
-                                }
-                            }
-                        }
-                    ],
-                    "topic":"YOUR_TOPIC",
-                    "bootstrapServers":"YOUR_KAFKA_BOOTSTRAP_SERVERS",
-                    "format":{
-                        "type":"jsonFormat",
-                        "failOnMissingField":false,
-                        "ignoreParseErrors":true,
-                        "timestampFormatStandard":"SQL",
-                        "mapNullKeyMode":"DROP",
-                        "mapNullKeyLiteral":"null",
-                        "encodeDecimalAsPlainNumber":true
-                    },
-                    "primaryKey":"id"
-                }
-            ],
-            "relations":[
-                {
-                    "type":"baseRelation",
-                    "inputs":[
-                        "1"
-                    ],
-                    "outputs":[
-                        "2"
-                    ]
-                }
-            ]
-        }
-    ]
-}
-```
-
-- Whole-database migration example：
+Step2. Install MySQL and turn on binlog. [MySQL Installation Guide](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/) and [MySQL binlog](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html)
```

Comment:
It is recommended to change to "Prepare a MySQL environment and enable binlog for it.  Please refer to [MySQL Installation Guide](https://dev.mysql.com/doc/mysql-installation-excerpt/5.7/en/) if there is no MySQL environment. Please refer to [MySQL Extract Load docs](xxx) about enable binlog for MySQL if the binlog is not enabled, of course it is also gotten directly refer to [MySQL binlog](https://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html)"